virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -111,6 +111,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-36162: Search option in VM configuration tab
 
 //CNV-36159: New configuration option "SSH over NodePort service"
+* A virtual machine's configuration section now allows *SSH over NodePort* as a service type.
```

Comment:
Can it be a bit more specific about the configuration of SSH over Nodeport?
Maybe should rephase it like the item below?